Airports/EGSC/Positions.txt

```diff
@@ -1,5 +1,2 @@
-EGSC_APP:Cambridge Radar:120.965:SCR:C:EGSC:APP:-:-:6160:6175:N052.12.18.000:E000.10.30.000
-EGSC_A_APP:Cambridge Approach:120.965:SCA:C:EGSC:APP:-:-:6160:6175:N052.12.18.000:E000.10.30.000
```

Comment:
Cambridge Approach (120.965) needs to be retained, just the director and radar are getting removed.